Editor/CoreAPI/CoreApi.cs

```diff
@@ -268,6 +278,21 @@ private static IAmazonS3Wrapper CreateDefaultS3Wrapper()
         {
             return new AmazonS3Wrapper(string.Empty, string.Empty, string.Empty);
         }
+        
+        public virtual string GetBucketPolicyPath()
+        {
+            string internalPath = $"{Paths.PackageName}/{Paths.BucketPolicyPath}";
+            string assetPath = $"Assets/{internalPath}";
+            string policyPath = Path.GetFullPath(assetPath);
+
+            if (!FileExists(policyPath))
+            {
+                string packagePath = $"Packages/{internalPath}";
+                policyPath = Path.GetFullPath(packagePath);
+            }
```

Comment:
Why is this condition required? Isn't the policy expected to be in the path you've defined below?

Reply:
I had copied the loading of cloud formationsn templates as a base, which seems to support it being in multiple locations, so I have removed this